CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## 0.12.2 - 2023-06-28
+
+## What's Changed
```

Comment:
Is this a convention somewhere else? I don't think we've ever had this header.

Reply:
Yeah that's a copy pasta mistake